src/app/view/mouseAndKeyboardHandler.js

```diff
@@ -198,6 +198,12 @@ export default class MouseAndKeyboardHandler {
                 reference.view.selectAllActive();
                 return false;
             }
+
+            if ((event.ctrlKey || event.metaKey) && key === 71) {
+                // Center the Design
+                reference.view.centerAll();
```

Comment:
We should use `paperview` class to be only for rendering-based tasks. The `centerAll()` function should be moved to `viewManager` (it would be `reference.centerAll()`)